tests/js/setup.ts

```diff
@@ -219,6 +211,89 @@ const routerFixtures = {
 };
 
 type TestStubTypes = TestStubFixtures & typeof routerFixtures;
+const jsFixturesDirectory = path.resolve(__dirname, '../../fixtures/js-stubs/');
+
+const extensions = ['.js', '.ts', '.tsx', '.json'];
+
+const specialMapping = {
+  AllAuthenticators: 'authenticators.js',
```

Comment:
hopefully, we can convert these files to `.tsx/.ts` one day 🙏

Reply:
Actually, I had a PR to change and import stubs directly just to try and out and there were a_lot of type errors. I suspect that as we make changes to these models in .ts code, we rarely forget to update the stubs and they fall out of sync...

Reply:
exactly! and it might happen that this will give us false, positive tests... I think its important to keep them updated and typescript can help us with that